<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="eth_src_mac_Edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>eth_src_mac:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="eth_dst_mac_Edit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>eth_dst_mac:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="arp_dst_mac_Edit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="arp_src_mac_Edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>arp_src_mac:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>arp_dst_mac:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>src_ip:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>dst_ip:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="src_ip_Edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dst_ip_Edit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="op_comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>request</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>require</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>frame:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frame_Edit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Send_pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
